src/pinning.js

```diff
@@ -320,13 +329,13 @@ class Pinning {
     this._openSubStores(address)
   }
 
-  _publish (type, odbAddress, data) {
+  async _publish (type, odbAddress, data) {
     const dataObj = { type, odbAddress }
     if (type === 'HAS_ENTRIES') {
       dataObj.numEntries = data
     } else if (type === 'REPLICATED') {
     }
-    this.pubsub.publish(this.pinningRoom, dataObj)
+    await this.pubsub.publish(this.pinningRoom, dataObj)
```

Comment:
Pretty sure this method doesn't return a promise.

Reply:
you're right, I was looking at the ipfs pubsub spec - looks like maybe orbitdb ignores the promise returned by ipfs?
ipfs spec: https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/PUBSUB.md#ipfspubsubpublishtopic-data
orbitdb wrapper: https://github.com/orbitdb/orbit-db-pubsub/blob/master/src/ipfs-pubsub.js#L74

Reply:
yeah, seems like it.